library/core/src/pin.rs

```diff
@@ -1003,22 +1003,26 @@ impl<P, U> CoerceUnsized<Pin<U>> for Pin<P> where P: CoerceUnsized<U> {}
 #[stable(feature = "pin", since = "1.33.0")]
 impl<P, U> DispatchFromDyn<Pin<U>> for Pin<P> where P: DispatchFromDyn<U> {}
 
-/// Constructs a <code>[Pin]<[&mut] T></code>, by pinning[^1] a `value: T` _locally_[^2].
+/// Constructs a <code>[Pin]<[&mut] T></code>, by pinning a value locally.
```

Comment:
Nit: since the API is macro-based, its exact type signature is hard to know. I don't think we should remove the input type annotation in this sentence unless we were to add the whole signature elsewhere:
```suggestion
/// Constructs a <code>[Pin]<[&mut] T></code>, by pinning a `value: T` _locally_.
```

If, on the other hand, we were to write something like:
```rs
fn pin<T>(value: T) -> Pin<&'local mut T>
```
then we could indeed keep your sentence as-is (although I personally like the emphasis on _locally_)

Reply:
Fair points about the type signature. I'm less sure about the emphasis on locally, as too much change in style hurts the read flow. The word is already quite prominent by virtue of being last in the sentence.